src/hub/Interfaces.container.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+  InterfaceRevision,
+  useListInterfacesFromInterfaceGroupQuery,
+} from "../generated/graphql";
+import InterfacesCard from "./InterfacesCard";
+import { ResourceReference } from "../wizard/ResourceRef";
+import InterfacesList from "./InterfacesList";
+
+export enum InterfacesView {
+  Card,
+  List,
+}
+
+export interface InterfaceRevisionWithKey extends InterfaceRevision {
+  key: string;
+}
+
+interface InterfacesContainerProps {
+  path: string;
+  view: InterfacesView;
+}
+
+function InterfacesContainer({ path, view }: InterfacesContainerProps) {
```

Comment:
It would be good to check whether a given InterfaceGroup actually exists (if it's as a whole `null`, or have 0 Interfaces assigned. Currently we can type anything in the URL and we can see a view which makes an impression that a given InterfaceGroup really exists:

![Screenshot 2022-01-11 at 15 02 18](https://user-images.githubusercontent.com/7155799/148957165-b296aa6b-f197-43a1-af43-c6defbc81db1.png)